drv/user-leds/src/main.rs

```diff
@@ -575,6 +537,100 @@ fn led_toggle(led: Led) {
     sys.gpio_toggle(pinset.port, pinset.pin_mask).unwrap_lite();
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// The STM32F4 specific bits.
```

Comment:
Take it or leave it: I wonder if we might want to move the board-specific bits into submodules with `#[cfg(feature = <board feature>]` on them, instead of sticking a `cfg` attribute on each function and constant? We could do the same for the existing F3 stuff. The code might read a little nicer that way. But, it's up to you.

Reply:
That sounds like a good follow-up PR. I just wanted to share what I'd done to make it run on the board I had.